.gitignore

```diff
@@ -39,3 +39,14 @@ yarn-error.log
 /.php_cs
 /.php_cs.cache
 ###< friendsofphp/php-cs-fixer ###
+
+### VisualStudioCode ###
```

Comment:
Would it suffice to just put `.vscode/*` in .gitignore? The `admin/` should def not be in here. :-)

Reply:
I have resolved this :-)